code/modules/clothing/masks/gasmask.dm

```diff
@@ -386,3 +386,25 @@
 	flags_inv = HIDEFACIALHAIR
 	w_class = WEIGHT_CLASS_NORMAL
 	inhand_icon_state = "driscoll_mask"
+
+/obj/item/clothing/mask/gas/prop
+	name = "prop gas mask"
+	desc = "A prop gas mask designed for appearance. Unlike a normal gas mask this does not filter gasses or protect against pepper spray."
+	icon_state = "gas_alt"
+	flags_inv = HIDEEARS|HIDEEYES|HIDEFACE|HIDEFACIALHAIR|HIDESNOUT
```

Comment:
Shouldn't have those if it's just a prop

Reply:
I'm a little confused are you saying I should remove those including the name and description?

Reply:
No wait not these, these are for sprite, resolve this one